src/App.tsx

```diff
@@ -1,61 +1,156 @@
 import './App.scss';
+import 'bulma/css/bulma.min.css';
+import classNames from 'classnames';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+const visibleTodos: Todo[] = todosFromServer
+  .map(todo => ({
+    ...todo,
+    user: getUser(todo.userId),
+  }));
+
+const todosId = visibleTodos.map(({ id }) => id);
 
 export const App = () => {
+  const [userId, setUserId] = useState(0);
+  const [title, setTitle] = useState('');
+  const [isCorrectUser, setIsCorrectUser] = useState(true);
+  const [isCorrectTitle, setIsCorrectTitle] = useState(true);
+
+  const getTodo = (): void => {
+    const newTodo: Todo = {
+      id: Math.max(...todosId) + 1,
```

Comment:
It doesn't work, because your todosId is static. Try to add couple todos and look their ids in DevTools.